CHANGELOG.md

```diff
@@ -77,6 +77,7 @@ Read more about the session argument for computed fields in the [docs](https://h
 - server: fix importing of allow list query from metadata (fix #4687)
 - server: flush log buffer during shutdown (#4800)
 - server: fix edge case with printing logs on startup failure (fix #4772)
+- console: add a flag for migrations on insert row section of schema page (#1766)
```

Comment:
this should perhaps be: `add a check box to flag an insertion as a migration on the schema page`